ncl/ncl_entries/trigonometric_functions.ipynb

```diff
@@ -0,0 +1,633 @@
+{
```

Comment:
<p>Coming back in with a few more thoughts I had last night.</p><p> </p><p>I think that it would make more sense to link to the ncl documentation here instead of the section heads, especially since sphinx is providing us with a floating section navigator bit on the right.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>